qutebrowser/config/configdata.yml

```diff
@@ -1141,7 +1141,7 @@ downloads.location.suggestion:
 completion.open_categories:
   type:
     name: FlagList
-    valid_values: [searchengines, quickmarks, bookmarks, history]
+    valid_values: [searchengines, quickmarks, bookmarks, history, filesystem]
     none_ok: true
   default:
     - searchengines
```

Comment:
Maybe add it as a default here (I'd say above `searchengines` even) for discoverability?

Reply:
I'm not sure. Opening files in a browser is pretty rare for me, so it feels off to put it at the top. Still, you probably won't notice this feature unless you see the category or read the release notes, so :shrug:

Reply:
Perhaps as a compromise, if we have a `completion.filesystem.favorites` (or whatever) setting, we could have that empty by default (i.e., not even including `~`)? Then the (empty) category is hidden when just doing `:open`, but still appears when starting to enter an absolute path.